### ISSUE TITLE
add aria-label to all instances of N / Total N displays

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1112]
N / Total N displays should have an aria-label applied to read out N out of Total N, otherwise it reads as N slash Total N in a screen reader.
